ios/browser/api/certificate/brave_certificate.h

```diff
@@ -0,0 +1,54 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_IOS_BROWSER_API_CERTIFICATE_BRAVE_CERTIFICATE_H_
+#define BRAVE_IOS_BROWSER_API_CERTIFICATE_BRAVE_CERTIFICATE_H_
+
+#import <Foundation/Foundation.h>
+#import <Security/Security.h>
+
+#if defined(BRAVE_CORE)
+#include "brave/ios/browser/api/certificate/common/brave_certificate_enums.h"
+#else
+#include <BraveCore/brave_certificate_enums.h>
+#endif
```

Comment:
Just do `#import "brave_certificate_enums.h"` and create a config in BUILD.gn that adds `common` to  `include_dirs`